src/player_team.ts

```diff
@@ -997,7 +997,22 @@ class Team {
       mults[i].base = pings[i].damage;
     }
 
-    let healing = 0;
+    // Apply poison damage.
+    let poison = 0;
+    for (const c of 'pm') {
+      const attr = COLORS.indexOf(c) as Attribute;
+      for (const combo of comboContainer.combos[c]) {
+        let multiplier = 0;
+        if (c == 'p') {
+          multiplier = 0.2 + (combo.count - 3) * 0.05;
+        } else if (c == 'm') {
+          multiplier = 0.5 + (combo.count - 3) * 0.125;
+        }
+        poison += this.getHp() * multiplier;
```

Comment:
I'm not sure, but doesn't poison damage round on each combo?

Reply:
Hmm. I dunno, but good catch, it definitely should round poison damage. I'll do some in-game tests and find out.

Reply:
Verified, rounds to the nearest integer.